model-mesh/runtimes/kustomization.yaml

```diff
@@ -26,8 +26,8 @@ images:
     newTag: 0.5.2
 
   - name: ovms-1
```

Comment:
that name is nowhere in the image field of https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/runtimes/ovms-1.x.yaml#L37 instead, there is a kustomize parameter value in there from https://github.com/opendatahub-io/odh-manifests/blob/master/model-mesh/base/params.env#L6 Same goes for the other three ServingRunetime yamls referenced ...